pom.xml

```diff
@@ -1050,7 +1050,7 @@
     </extensions>
 
     <plugins>
-      <plugin>
+     <!-- <plugin>
```

Comment:
why is it commented out ?

Reply:
There is api difference between our last netty version `siren-4.1.68-1` and the current version `siren-4.1.78-1-SNAPSHOT`, a new method `PooledByteBufAllocator::pinnedDirectMemory()` has been added.

```
[INFO] Comparing [io.netty:netty-buffer:jar:siren-4.1.68-1] against [io.netty:netty-buffer:jar:siren-4.1.78-1-SNAPSHOT] (including their transitive dependencies).
[INFO] API problems found.
[ERROR] Failed to execute goal org.revapi:revapi-maven-plugin:0.14.6:check (api-check) on project netty-buffer: The following API problems caused the build to fail:
[ERROR] java.method.finalMethodAddedToNonFinalClass: method long io.netty.buffer.PooledByteBufAllocator::pinnedDirectMemory(): A final method has been added to an inheritable class. https://revapi.org/revapi-java/differences.html#java.method.finalMethodAddedToNonFinalClass
```

Reply:
fixed with 52fbae0d6e65c72ea8a8b57c1ec9c33c8d89f98d